### PR TITLE
added 2022-06-09 weekly meeting details

### PR DESCRIPTION
**Pull request by Ayush7614**
Signed-off-by: Ayush Kumar [ayushknj3@gmail.com](mailto:ayushknj3@@gmail.com)

## Project

ReactUI

### Changes

Added 9 June Meeting update

### CODE REVIEW

File: docs/2022/ui/updates/ayush/2022-06-09.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- [Rohit Pandey](https://github.com/rohitpandey49)
+- [Thanvi Lahari Pendyala](https://github.com/Pendyala-thanvi)
+- [Ayush Kumar](https://github.com/Ayush7614)
+- [Krishna Mahato](https://github.com/krishna9304)
+- [Soham Banerjee](https://github.com/soham4abc)
+
+### Discussions
+
+- Fossology Setup Done
+- My Sem end exams are going on then after i start contribution
```
Comment (GMishx) [inline docs/2022/ui/updates/ayush/2022-06-09.md:41]:
```suggestion
- My Sem end exams are going on, after which I start to contribution.
```

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2022/ui/updates/ayush/2022-06-09.md (hunk 1 above)
Comment (Ayush7614) [reply docs/2022/ui/updates/ayush/2022-06-09.md]: Done @GMishx

**Comment by Ayush7614** [comment]
migrate all weekly meeting code and details in one file

File: docs/2022/ui/updates/ayush/2022-06-02.md [hunk 2]
```diff
@@ -55,4 +55,100 @@ GSOC 2022 weekly update.
...
+---
+
+<!--
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2022 Ayush Kumar <ayushknj3@gmail.com>
+-->
+
+## Meeting 3
+*(June 16th, 2022)*
```
Comment (GMishx) [inline docs/2022/ui/updates/ayush/2022-06-02.md:123]: Hello @Ayush7614 ,
As from June 13, the coding period started, so please move updates to a new weekly file under same folder as done in previous run:
https://github.com/fossology/gsoc/tree/main/docs/2021/copyrights/updates

Comment (GMishx) [reply docs/2022/ui/updates/ayush/2022-06-02.md:123]: Any updates on this @Ayush7614 ?

**Comment by GMishx** [comment]
Can we close the PR @Ayush7614 if there are no updates from you?

**Comment by Ayush7614** [comment]
No @GMishx wait for today i am update all the reports and added all the meeting discussion in my next PRs and also correct this PR

**Comment by GMishx** [comment]
Closing the PR as there are no updates.

Please reopen if needed.

**Pull request closed without merging**